logging/bin/getlogs.py

```diff
@@ -9,22 +9,34 @@
 import os, sys
 import json, csv
 
-## Cleanup temp file if not previously deleted
-if os.path.isfile("temp_SAS_OSQ.TXT"):
-    os.remove("temp_SAS_OSQ.TXT")
+## Cleanup temp file if not previously deleted, Create target directory
+if not os.path.isdir("GetLogs_Files"):
+    os.mkdir("GetLogs_Files")
+
+if os.path.isfile("GetLogs_Files/temp_SAS_OSQ.TXT"):
+    os.remove("GetLogs_Files/temp_SAS_OSQ.TXT")
 
 def validate_input(dict):
     """Validate the arguments passed by the user to ensure script will function properly"""
+    
+    if(len(sys.argv) == 1): ##Check if any args have been provided
+        print("No arguments have been provided")
+        exit()
+
     if dict['create-auth']:
         if (type(dict['create-auth'] == list)):
             dict['create-auth'] = " ".join(dict['create-auth'])
+        if (not "GetLogs_Files/" in  dict['create-auth']):
```

Comment:
A little confused about the need for this check. Is the user expected to include the directory name with the file name? If so, what will happen if they are passing in a different directory name? Maybe its fine but just wanted to pose the question.